view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -281,6 +281,23 @@ define(
                     self.threeDS2ChallengeComponent.mount(threeDS2Node);
                 }
             },
+            threedsfallback: function (action) {
+                var self = this;
+                var actionNode = document.getElementById('ActionContainer');
+
+                self.popupModal = $('#ActionModal').modal({
```

Comment:
@AlexandrosMor I haven't really realised it until now that this modal is not necessary at all. We don't show anything to the customer in the modal, right?